x-pack/plugins/reporting/export_types/printable_pdf/server/execute_job/index.js

```diff
@@ -6,46 +6,24 @@
 
 import url from 'url';
 import * as Rx from 'rxjs';
-import { mergeMap, catchError, map, takeUntil } from 'rxjs/operators';
-import { omit } from 'lodash';
+import { mergeMap, map, takeUntil } from 'rxjs/operators';
 import { UI_SETTINGS_CUSTOM_PDF_LOGO } from '../../../../common/constants';
 import { oncePerServer } from '../../../../server/lib/once_per_server';
 import { generatePdfObservableFactory } from '../lib/generate_pdf';
-import { cryptoFactory } from '../../../../server/lib/crypto';
 import { compatibilityShimFactory } from './compatibility_shim';
 
-const KBN_SCREENSHOT_HEADER_BLACKLIST = [
```

Comment:
Other than the authorization/cookie headers, were there any other headers that we might have relied on? **If** I understand the change correctly, instead of passing all non-blacklisted headers through, we are instead only supplying the session cookie. I agree this is what we need for auth, but I'm worried that we were unknowingly using another header that won't be present anymore. For example, controlling for language (i18n), timezone, or any X-* headers that a reverse proxy might be expecting

Reply:
I can't think of any that we'd want to be forcing. When the user visits Kibana using their own browser, they don't force any headers so I can't think of any reason why we'd want to either.